encord/project.py

```diff
@@ -375,6 +376,12 @@ def add_classification(
         """
         return self._client.add_classification(name, classification_type, required, options)
 
+    def list_models(self) -> List[ModelConfiguration]:
```

Comment:
If we want to be consistent with, e.g., datasets and label rows, this should probably be a property called `models` instead. What do you think @denis-cord?

Reply:
I think that is a good point. 

What I was thinking about is that we might want to introduce a filter at some point like we do in `list_label_rows`. In that case we could add the filter methods in a new non-property accessor. 

So I guess the consideration is:
* do we believe that we might want to have filters at some point for models (I think it may be possible)
* in that case - would we rather have two different methods or one (I thought one would be a bit nicer).

But I am also happy to have only the property accessor for now if you say that would seem more usable.